### DISCUSSION TITLE
How to edit node's dialog text field in GUI

### DISCUSSION THREAD

**Discussion by theseusyang1** [enso-org/enso#7118] [Help and troubleshooting]
<img width="605" alt="image" src="https://github.com/enso-org/enso/assets/113496312/eb60bff0-cf58-4647-9d1b-0339f0f9cf7e">

how to edit node's dialog text field , I can't find it!

**Comment by wdanilo** [comment]
What do you mean by "node's dialog"? The box below node is the output visualization, it is not editable. If you want to edit the text on nodes, you can now `ctlr/cmd + LMB` click on the node to edit it. Does it answer your question? :)
